environment.yml

```
@@ -7,13 +7,7 @@ channels:
+  - pyopenms
   - pandas
-  - numpy
-  - pyarrow
-  - ms2rescore=3.0.3
-  - deepLC=2.2.38
-  - psm-utils=0.8.3
-  - scipy=1.13.1
-  - pygam
-  - protobuf=3.19.6
+  - pyarrow>=18.0.0
+  - scipy
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)